### PR TITLE
Better TypeScript Prompts

### PR DESCRIPTION
**Pull request by neelsudhakaran**
### Changes

Renamed "Help Me Choose" option when selecting a TypeScript setup to "I'm Unsure" as "Help Me Choose" was misleading (See #5352).

### Testing

I just manually ran the CLI and saw if everything looked good (which I think it did)!

### Docs

This is such a minor change no docs changes should be needed!

### Comments

In #5352, we discussed about naming it "Use Default", but I think that having it as "I'm Unsure" fits the resulting action even better! Let me know your thoughts on this though.

### CODE REVIEW

**Review by Princesseuh — APPROVED** [review]

File: packages/create-astro/src/index.ts
Comment (Princesseuh) [inline packages/create-astro/src/index.ts:293]:
```suggestion
				{ value: 'unsure', title: `I'm unsure` },
```

**Comment by Princesseuh** [comment]
Just suggested a small change, otherwise, it looks good to me. Thank you for contributing!

**Comment by Astisme** [comment]
I'm not a fan of this default. I've made a do/while that sends a link to the docs to the user then asks again for the choice. May I publish it?

**Comment by Princesseuh** [comment]
> I'm not a fan of this default. I've made a do/while that sends a link to the docs to the user then asks again for the choice. May I publish it?

Sure, feel free to open a PR with your changes! We'll probably discuss it a bit with the team to evaluate which solution works best.

**Comment by neelsudhakaran** [comment]
In regards to the force push, I haven't really done much open source contributing before, so I thought it was better to undo a merge commit than to leave something I totally didn't understand and that might break things!

**Comment by matthewp** [comment]
Thank you for the PR! We've talked about it with the maintainers and given that there are a couple of proposed solutions for this (https://github.com/withastro/astro/pull/5368 being the other), we've decided that we would rather just remove the question and use `relaxed` by default. So I'm going to close this PR, but thank you so much for taking the time to create it.

**Pull request closed without merging**
